setup.py

```diff
@@ -244,8 +244,19 @@ def run(self):
 
 # Qt bindings requirements
 qt_requirements = {
-    'pyqt5': ['pyqt5>=5.15,<5.16', 'pyqtwebengine>=5.15,<5.16'],
-    'pyqt6': ['pyqt6>=6.5,<7', 'pyqt6-webengine>=6.5,<7'],
+    'pyqt5': [
+        'pyqt5>=5.15,<5.16',
+        'pyqtwebengine>=5.15,<5.16',
+        'qtconsole>=5.6.1,<5.7.0',
+    ],
+    'pyqt6': [
+        'pyqt6>=6.5,<7',
+        'pyqt6-webengine>=6.5,<7',
+        'qtconsole>=5.6.1,<5.7.0',
+    ],
+    'none': [
```

Comment:
```suggestion
    'conda-forge': [
```

Let's change this to `conda-forge` to be quite explicit about this env value (also because `qtconsole-base` doesn't exist on PyPI).